### PR TITLE
#1 bootstrap project

### PR DESCRIPTION
**Pull request by jchabrowski**
#1 Project bootstrap

### CODE REVIEW

File: frontend/src/input.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (jchabrowski) [inline frontend/src/input.css:3]: 🧲

File: frontend/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet">
+    <title>coach-panel-ap</title>
```
Comment (lrozanski) [inline frontend/index.html:7]: coach-panel-app

File: frontend/.gitignore [hunk 3]
```diff
@@ -1,24 +1,8 @@
...
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-lerna-debug.log*
+# dotenv
+.env
+.env.development.local
+.env.test.local
+.env.production.local
+.env.local
```
Comment (lrozanski) [inline frontend/.gitignore:6]:
```
.env*
```
Would be simpler, and would catch other cases

File: frontend/.gitignore [hunk 4]
```diff
@@ -1,24 +1,8 @@
...
-.vscode/*
-!.vscode/extensions.json
-.idea
-.DS_Store
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
+/node_modules
```
Comment (lrozanski) [inline frontend/.gitignore:8]: `node_modules` instead of `/node_modules`

File: frontend/.prettierrc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 90,
+  "tabWidth": 2,
+  "semi": true,
```
Comment (lrozanski) [inline frontend/.prettierrc:4]: Let's switch this to false, and use spaces instead of tabs

File: frontend/dist/output.css [hunk 6]
```diff
@@ -0,0 +1,586 @@
+/*
```
Comment (lrozanski) [inline frontend/dist/output.css:1]: Add this folder to `.gitignore`

**Comment by lrozanski** [comment]
Fixes #1

**Review by lrozanski — APPROVED** [review]

**Pull request merged**